### PR TITLE
Updated doc to state Cluster Autoscaler is supported

### PR DESCRIPTION
**Pull request by TheRealGoku**
Explicitly state the Cluster Autoscaler is supported as the VPC endpoint requirements satisfied without any limitation

### CODE REVIEW

File: doc_source/private-clusters.md [hunk 1]
```diff
@@ -69,4 +70,4 @@ The following [VPC endpoints](https://docs.aws.amazon.com/vpc/latest/userguide/v
 + `com.amazonaws.<region>.sts` – If using AWS Fargate or IAM roles for service accounts
```
Comment (buzzsurfr) [inline doc_source/private-clusters.md:70]: Need to add "or cluster autoscaler" to this line.

**Review by buzzsurfr — COMMENTED** [review]
One minor change for clarity, then I'll approve.

File: doc_source/private-clusters.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Here are some things to consider when running Amazon EKS in a private cluster wi
 + [App Mesh](https://docs.aws.amazon.com/app-mesh/latest/userguide/what-is-app-mesh.html) is supported with private clusters when you use the App Mesh Envoy VPC endpoint\. For more information, see [VPC endpoints for private clusters](#vpc-endpoints-private-clusters)\.
   + The App Mesh sidecar injector for Kubernetes is supported\. For more information, see [App Mesh sidecar injector](https://github.com/aws/aws-app-mesh-inject) on GitHub\.
   + The App Mesh controller for Kubernetes is not supported\. For more information, see [App Mesh controller](https://github.com/aws/aws-app-mesh-controller-for-k8s) on GitHub\.
+  + The [Cluster Autoscaler](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html) is supported\. You must include the STS VPC endpoint and Autoscaling VPC endpoint\. For more information, see [VPC endpoints for private clusters](#vpc-endpoints-private-clusters)\
```
Comment (buzzsurfr) [inline doc_source/private-clusters.md:32]: Also add the fix as described in https://github.com/kubernetes/autoscaler/issues/3992

**Review by buzzsurfr — COMMENTED** [review]
One more change

File: doc_source/private-clusters.md (hunk 1 above)
Comment (TheRealGoku) [reply doc_source/private-clusters.md]: Updated and did [commit](https://github.com/awsdocs/amazon-eks-user-guide/pull/374/commits/e93c03fc0f541726551d04b65591961a97301c2c) under this same pull request

File: doc_source/private-clusters.md (hunk 2 above)
Comment (TheRealGoku) [reply doc_source/private-clusters.md]: Added the clause and reframed the sentence. [Commit](https://github.com/awsdocs/amazon-eks-user-guide/pull/374/commits/d6f7628eae2e93ec6a21fa423cc7ee5e0678342a)

**Comment by pgasca** [comment]
Thank you @TheRealGoku and @buzzsurfr I've completed merging various inputs and feedback regarding this. Feel free to do another pull request if you notice anything else.

**Pull request closed without merging**